other/materialsproject/import_materials_from_materialsproject.py

```
@@ -44,7 +44,9 @@
         )
     )
 
-    get_ipython().system('GIT_BRANCH="dev"; export GIT_BRANCH; curl -s "https://raw.githubusercontent.com/Exabyte-io/api-examples/${GIT_BRANCH}/scripts/env.sh" | bash')
+    get_ipython().system(
```

Ruff (0.15.12)

[error] Undefined name `get_ipython`

(F821)